review-me/review-me.js

```diff
@@ -0,0 +1,31 @@
+// hello.js
+
+var message = "Hello, world!"
+console.log(MESSAGE);
+
+function printmessage(msg) {
+    if(msg != null){
+        console.log(msg)
+    }
+    else{
+        console.log("Nothing to print")
+    }
+}
+
+printmessage(message)
+
+eval("console.log('Eval should not be used!')")
```

Comment:
17  :  Usage  of  eval  can  lead  to  security  vulnerabilities  and  unpredictable  behavior